src/mmw/apps/modeling/models.py

```diff
@@ -79,7 +79,7 @@ class Meta:
     name = models.CharField(
         max_length=255)
     project = models.ForeignKey(Project, related_name='scenarios')
-    current_condition = models.BooleanField(
+    is_current_conditions = models.BooleanField(
```

Comment:
I thought it would make the app a little more flexible in the future. We could easily add the ability to run a different models per scenario in the same project if we just specify the model simulation name here. For "current condition" we could just have a null or noop model simulation?